aissemble-open-inference-protocol-examples/aissemble-oip-kserve/pom.xml

```
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.boozallen.aissemble</groupId>
+        <artifactId>aissemble-open-inference-protocol-examples</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>aissemble-oip-kserve-inference-fastapi</artifactId>
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>aissemble-oip-kserve-inference-fastapi</module>
+    </modules>
+
+    <name>aiSSEMBLE::Open Inference Protocol Examples KServe for FASTAPI</name>
```

s: This is the parent module for all kserve examples. Not just this one
```suggestion
    <name>aiSSEMBLE::Open Inference Protocol Examples::KServe</name>
```